test/commands/auth/scratch-idenfity.nut.ts

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
Typo in test file name `identify`. Maybe you meant `identify`?

Reply:
💩